Tests/ArcGISToolkitTests/BookmarksTests.swift

```diff
@@ -59,9 +56,10 @@ final class BookmarksTests: XCTestCase {
             get: { _isPresented },
             set: {_isPresented = $0 }
         )
+        
+        var actionBookmark: Bookmark?
         let action: ((Bookmark) -> Void) = {
```

Comment:
Declaring a variable for this isn't necessary. Also, since the stored value is the selected bookmark, I would suggest calling the variable that stores the bookmark `selectedBookmark`. In addition, just compare the bookmarks themselves rather than their viewpoints. Lastly, `bookmarks` doesn't need to be mutable. It would be better to test using the public modifier. Putting that all together:

```swift
var selectedBookmark: Bookmark?
let bookmarks = Bookmarks(isPresented: isPresented,geoModel: map)
    .onSelectionChanged { selectedBookmark = $0 }
XCTAssertTrue(_isPresented)
let firstBookmark = try XCTUnwrap(map.bookmarks.first)
bookmarks.selectBookmark(firstBookmark)
XCTAssertFalse(_isPresented)
XCTAssertEqual(selectedBookmark, map.bookmarks.first)
```

Reply:
Good stuff, thanks!

Reply:
You're welcome!